java/ingest/ingest-batcher-submitter/src/test/java/sleeper/ingest/batcher/submitter/IngestBatcherSubmitterLambdaIT.java

```diff
@@ -258,6 +265,7 @@ void shouldIgnoreAndLogMessageWithInvalidJson() {
 
             // Then
             assertThat(store.getAllFilesNewestFirst()).isEmpty();
+            assertThat(receiveDeadLetters()).isNotEmpty();
```

Comment:
It looks like these tests need renaming a bit to reflect the change in behaviour.

We also need at least one test that asserts that the dead letter queue is empty. It looks like shouldIgnoreMessageIfFilesNotProvided should assert that? Maybe we should also assert that in one of the happy path tests?